reledmac.dtx

```diff
@@ -428,10 +428,10 @@
 % \changes{v2.15.1}{2016/09/07}{Fix \protect\cs{edindex} in tabular environments.}
 % \changes{v2.15.2}{2016/09/12}{Fix bug with \protect\package{fancyhdr} package 3.8 and later.}
 % \changes{v2.15.3}{2016/09/26}{Fix bug with \protect\cs{section} in optional argument of \protect\cs{pstart} and empty line before \protect\cs{pend} (bug added in v2.8.2).}
-% \changes{v2.15.3}{2016/09/26}{Simplification of the code concerning sectioning command.}
-% \changes{v2.16.0}{2016/10/01}{Display warning message if using a too older versions of \protect\LaTeX.}
+% \changes{v2.15.3}{2016/09/26}{Simplification of the sectioning command code.}
+% \changes{v2.16.0}{2016/10/01}{Display warning message if using a version of \protect\LaTeX that is too old.}
 % \changes{v2.16.0}{2016/10/01}{Deleting dead code.}
-% \changes{v2.16.0}{2016/10/01}{New options of \protect\cs{fnpos} and \protect\cs{mpfnpos} to set a customized alternance between familiar and critical footnotes.}
+% \changes{v2.16.0}{2016/10/01}{New options of \protect\cs{fnpos} and \protect\cs{mpfnpos} to set a customized alternative between familiar and critical footnotes.}
```

Comment:
" to set a customized order for familiar and critical footnotes"… I think that will be enough ;)